cmd/grype/cli/options/grype.go

```diff
@@ -11,30 +11,31 @@ import (
+	OnlyNotFixed                           bool               `yaml:"only-notfixed" json:"only-notfixed" mapstructure:"only-notfixed"`                      // only fail if detected vulns don't have a fix
+	IgnoreStates                           string             `yaml:"ignore-states" json:"ignore-wontfix" mapstructure:"ignore-wontfix"`                    // ignore detections for vulnerabilities matching these comma-separated fix states
+	Platform                               string             `yaml:"platform" json:"platform" mapstructure:"platform"`                                     // --platform, override the target platform for a container image
+	Search                                 search             `yaml:"search" json:"search" mapstructure:"search"`
+	Ignore                                 []match.IgnoreRule `yaml:"ignore" json:"ignore" mapstructure:"ignore"`
+	Exclusions                             []string           `yaml:"exclude" json:"exclude" mapstructure:"exclude"`
+	DB                                     Database           `yaml:"db" json:"db" mapstructure:"db"`
+	ExternalSources                        externalSources    `yaml:"external-sources" json:"externalSources" mapstructure:"external-sources"`
+	Match                                  matchConfig        `yaml:"match" json:"match" mapstructure:"match"`
+	FailOn                                 string             `yaml:"fail-on-severity" json:"fail-on-severity" mapstructure:"fail-on-severity"`
+	Registry                               registry           `yaml:"registry" json:"registry" mapstructure:"registry"`
+	ShowSuppressed                         bool               `yaml:"show-suppressed" json:"show-suppressed" mapstructure:"show-suppressed"`
+	ByCVE                                  bool               `yaml:"by-cve" json:"by-cve" mapstructure:"by-cve"` // --by-cve, indicates if the original match vulnerability IDs should be preserved or the CVE should be used instead
+	Name                                   string             `yaml:"name" json:"name" mapstructure:"name"`
+	DefaultImagePullSource                 string             `yaml:"default-image-pull-source" json:"default-image-pull-source" mapstructure:"default-image-pull-source"`
+	VexDocuments                           []string           `yaml:"vex-documents" json:"vex-documents" mapstructure:"vex-documents"`
+	VexAdd                                 []string           `yaml:"vex-add" json:"vex-add" mapstructure:"vex-add"`                                                                                                             // GRYPE_VEX_ADD
+	ShowKernelHeadersMatchedUpstreamKernel bool               `yaml:"show-kernel-headers-matched-upstream-kernel" json:"show-kernel-headers-matched-upstream-kernel" mapstructure:"show-kernel-headers-matched-upstream-kernel"` // Show matches on kernel-headers packages where the match is on kernel upstream instead of marking them as ignored, default=false
```

Comment:
The reason I was looking to avoid the term "show" is that the `show-suppressed` is actually only for the presentation in the table view whereas this is excluding matches, affecting other parts like `--fail-on`.

How about calling this `MatchUpstreamKernelHeaders`?

Reply:
I'll make that change asap.